universal-login-sdk/test/core/services/WalletSerializer.test.ts

```diff
@@ -206,7 +206,7 @@ describe('UNIT: WalletSerializer', () => {
       });
 
       expect(state.kind).to.eq('Deployed');
-      expect((state as any).wallet).to.deep.include(TEST_APPLICATION_WALLET);
+      expect((state as any).wallet).to.deep.include({...TEST_APPLICATION_WALLET, email: undefined});
```

Comment:
same

Reply:
Here is not necessary, but it is just a sanity check that email property is included here with undefined value